evaluator/evaluator.go

```diff
@@ -138,11 +203,17 @@ func evalSelectStatement(backend Backend, ss *ast.SelectStatement) object.Object
 		}
 		rowsToReturn = append(rowsToReturn, row)
 	}
+
+	/// 3) Post-processing: Populate aliases, sort, and limit
 	// Populate aliases
 	for i, alias := range ss.Aliases {
 		if alias == "" {
 			aliases[i] = ss.Expressions[i].String()
 		}
+		// remove table name from alias (if present)
+		if strings.Contains(aliases[i], ".") {
+			aliases[i] = strings.Split(aliases[i], ".")[1]
+		}
```

Comment:
This should not be necessary. If identifier evaluation is refactored, maybe this is fixed in the process